tests/beta/queuing/test_beta_operation.py

```diff
@@ -1,88 +0,0 @@
-# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
Is this to be introduced again in a later episode?

Reply:
Nope; this beta tensor object has been replaced by the monkeypatching of the regular tensor.